dev-libs/svrcore/svrcore-4.1.2.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit multilib autotools
```

Comment:
Because the original ebuild author did some time ago.  I'm just trying to get a version bump through and wasn't aware of this fact.